politeiawww/cmd/piwww/piwww.go

```diff
@@ -45,62 +45,67 @@ type piwww struct {
+	UserEdit           userEditCmd             `command:"useredit" description:"(user) edit the preferences of the logged in user"`
+	UserNew            userNewCmd              `command:"usernew" description:"(public) create a new user"`
+	UserPaymentsRescan userPaymentsRescanCmd   `command:"userpaymentsrescan" description:"(admin) rescan a user's payments to check for missed payments"`
+	UserPendingPayment userPendingPaymentCmd   `command:"userpendingpayment" description:"(user) get details for a pending payment for the logged in user"`
+	UserProposals      userProposalsCmd        `command:"userproposals" description:"(public) get all proposals submitted by a specific user"`
+	UserVerifyEmail    userVerifyEmailCmd      `command:"userverifyemail" description:"(public) verify a user's email address"`
+	UserVerifyPayment  userVerifyPaymentCmd    `command:"userverifypayment" description:"(user) check if the logged in user has paid their user registration fee"`
+	Users              shared.UsersCmd         `command:"users" description:"(public) get a list of users"`
+	UserManage         shared.UserManageCmd    `command:"usermanage" description:"(admin) edit certain properties of the specified user"`
+	UserUpdateKey      shared.UserUpdateKeyCmd `command:"userupdatekey" description:"(user) generate a new identity for the logged in user"`
+
+	// XXX will be factored to a user route
+	ProposalPaywall proposalPaywallCmd `command:"proposalpaywall" description:"(user)   get proposal paywall details for the logged in user"`
+
+	// Basic commands
 	Login              shared.LoginCmd          `command:"login" description:"(public) login to Politeia"`
 	Logout             shared.LogoutCmd         `command:"logout" description:"(public) logout of Politeia"`
-	ManageUser         shared.ManageUserCmd     `command:"manageuser" description:"(admin)  edit certain properties of the specified user"`
-	Me                 shared.MeCmd             `command:"me" description:"(user)   get user details for the logged in user"`
-	NewUser            NewUserCmd               `command:"newuser" description:"(public) create a new user"`
-	Policy             PolicyCmd                `command:"policy" description:"(public) get the server policy"`
-	ProposalPaywall    ProposalPaywallCmd       `command:"proposalpaywall" description:"(user)   get proposal paywall details for the logged in user"`
-	RescanUserPayments RescanUserPaymentsCmd    `command:"rescanuserpayments" description:"(admin)  rescan a user's payments to check for missed payments"`
-	ResendVerification ResendVerificationCmd    `command:"resendverification" description:"(public) resend the user verification email"`
+	Me                 shared.MeCmd             `command:"me" description:"(user) get user details for the logged in user"`
+	ChangePassword     shared.ChangePasswordCmd `command:"changepassword" description:"(user) change the password for the logged in user"`
```

Comment:
Let's make these command names consistent with the www routes.  
https://github.com/lukebp/politeia/blob/tlog/politeiawww/api/www/v1/v1.go#L31-L51

I think the one exception should be the me command.  So `login`, `logout`, and `me` will all be kept the same.  The other user commands will start with User.

Examples:
```
UserPasswordChange
UserUsernameChange
UserTOTPSet
UserTOTPVerify
...
```